yaramo/node.py

```diff
@@ -120,6 +120,13 @@ def get_arc_between_nodes(_node_a: "Node", _node_b: "Node"):
         else:
             self.connected_on_left, self.connected_on_right = other_a, other_b
 
+    def is_switch(self):
```

Comment:
we are using "point" as term for a switch everywhere else in yaramo, so it would be good to use that here too.

Apart from that, we had a lot of discussion about the datastructure of Node vs. Point. Currently, we are treating nodes as points unless they only have one other node connected, as you correctly used in this function. Our idea would be that we could also introduce subclasses of Node (Point and End) that allow for the common features (navigating the graph) and also only offer attributes that only points have (connected_on_left etc.) when appropriate. As this would be a major refactoring across all repositories and there is antother possibly even bigger refactoring waiting in https://github.com/simulate-digital-rail/yaramo/issues/31, we didn't do that yet. Maybe you can discuss among yourselves or with @arneboockmeyer whether that could be something that you want to tackle

Reply:
I wasn't aware of that assumption, thanks! My take would be that until the refactoring is implemented (or the further plan is clarified), we should make our lives still easier by centralizing the decision logic if a node is a `switch` or a `end`.